src/main.py

```diff
@@ -89,14 +95,22 @@ def transcribe():
             if request_is_invalid:
                 return request_is_invalid
 
-            result = transcriber.transcribe(
-                file=tempFile,
-                requestedModel=request.args.get("model", DEFAULT_MODEL),
-                task=request.args.get("task", DEFAULT_TASK),
-                output=request.args.get("output", DEFAULT_OUTPUT),
-                language=request.args.get("language")
+            filename = tempFile.name
+            requestedModel = request.args.get("model", DEFAULT_MODEL)
+            task = request.args.get("task", DEFAULT_TASK)
+            output = request.args.get("output", DEFAULT_OUTPUT)
+            language = request.args.get("language")
+
+            job = rq_queue.enqueue(
+                'transcriber.transcribe',
+                args=(filename,requestedModel,task,output,language),
+                result_ttl=3600*24*7,
+                on_success=mailer.send_success_email,
+                on_failure=mailer.send_failure_email
             )
 
+            return 'Bananese',201
```

Comment:
Should this return any data? Would seem better to return a 201 response with the queue task ID maybe?

also the code below does not seem to be reachable